tmt/steps/discover/fmf.py

```diff
@@ -260,6 +268,25 @@ class DiscoverFmf(tmt.steps.discover.DiscoverPlugin[DiscoverFmfStepData]):
     Note that internally the modified tests are appended to the list
     specified via ``test``, so those tests will also be selected even if
     not modified.
+
+    To modify the discovered tests' metadata directly from the plan (for example if
+    it is an yet-to-be added package, special HW, or missing permissions to modify the tests repo),
+    you can use the ``adjust-tests``. Its value is the list of actions to do, each action is the
+    "merging" operation of fmf (similar form as is used in the adjust rules)
+
+    Following example adds an 'avc' check for each discovered test, doubles its duration and
+    replaces each occurrence of the word 'python3.11' in the list of required packages.
+
+    .. code-block:: yaml
+
+        discover:
+            how: fmf
+            adjust-tests:
+            - check+:
+                - how: avc
+            - duration+: '*2'
+            - require~:
+                - '/python3.11/python3.12/'
```

Comment:
If `when` and `description` should work, I'd like to see an example that uses them.

Reply:
extended in 3f12f70e